core-image.html

```diff
@@ -202,7 +202,7 @@
       if (this.preload) {
         if (this.fade) {
           if (!this._placeholderEl) {
-            this._placeholderEl = this.shadowRoot.querySelector('#placeholder');
+            this._placeholderEl = this.shadowRoots["core-image"].querySelector('#placeholder');
```

Comment:
Prefer using the [`this.$` syntax](https://www.polymer-project.org/docs/polymer/polymer.html#automatic-node-finding) when accessing the shadow DOM:

    this._placeholderEl = this.$.placeholder;